frontend/src/routes/Governance/components/ViewDiffApiCall.tsx

```diff
@@ -0,0 +1,102 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+import { useEffect, useState } from 'react'
+import { fireManagedClusterView } from '../../../resources'
+import { resultsTableData } from '../policies/policy-details/PolicyDetailsResults'
+import { useTranslation } from '../../../lib/acm-i18next'
+import { Alert, Button, Divider, Modal, ModalVariant, Skeleton, Title } from '@patternfly/react-core'
+import Highlight from 'react-highlight'
+import './github.min.css'
+import { Grid } from '@mui/material'
+
+export function ViewDiffApiCall({ item }: Readonly<{ item: resultsTableData }>) {
+  const { templateName, apiVersion, kind, cluster } = item
+  const [templateErr, setTemplateError] = useState<string>()
+  const [isFetching, setIsFetching] = useState<boolean>(false)
+  const [relatedObjs, setRelatedObjs] = useState<any[]>()
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  const { t } = useTranslation()
+  const handleModalToggle = () => {
+    setIsModalOpen((prevIsModalOpen) => !prevIsModalOpen)
+  }
+  useEffect(() => {
+    if (kind === 'IamPolicy' && apiVersion.includes('policy.open-cluster-management.io')) {
```

Comment:
Is this check required since "ViewDiffApiCall" should only be called for a ConfigurationPolicy?